format/Flight.proto

```diff
@@ -301,6 +301,33 @@ message Location {
  */
 message Ticket {
   bytes ticket = 1;
+  // Data representing some part of the data retrievable by the ticket.
+  //
+  // `inlined_completeness` indicates what part of the data retriavable
+  // by the ticket this represents. This is provided as an optimization for
+  // client/server applications that want to reduce latency to first result
+  // without requiring another RPC round-trip to retrieve the ticket.  applications
+  // built on top of Flight are responsible for any negotiation necessary on whether
+  // inlining data is appropriate.
+  //
+  // The size of inlined_data is expected to be small (typically less then 1MB) 
+  // and inlining too much data across tickets can run into underlying transport 
+  // limitations.  Furthermore, since the data is expected to be small, implementations 
+  // are less likely to optimize for zero-copy in these cases.  
+  repeated FlightData inlined_data = 2;
```

Comment:
Do we expect servers to repeat the schema message, or to assume the schema is identical to the one in FlightInfo, and that this will contain only dictionary batches and record batches?

Reply:
I was thinking same schema as flight info but hadn't thought deeply about this.   Do you have a preference?

Reply:
I would lean towards the same schema for consistency, but either way it should just be clearly spelled out